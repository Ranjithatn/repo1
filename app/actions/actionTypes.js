// Auth Actions
export const REQUEST_LOGIN = "REQUEST_LOGIN";
export const REQUEST_LOGIN_SUCCESS = "REQUEST_LOGIN_SUCCESS";
export const REQUEST_LOGIN_FAILED = "REQUEST_LOGIN_FAILED";
export const REQUEST_LOGOUT = "REQUEST_LOGOUT";
export const REQUEST_LOGOUT_SUCCESS = "REQUEST_LOGOUT_SUCCESS";
export const REQUEST_LOGOUT_FAILED = "REQUEST_LOGOUT_FAILED";
export const REQUEST_SET_NEW_UPDATE = "REQUEST_SET_NEW_UPDATE";
export const REQUEST_SHOW_UPDATE_MESSAGE = "REQUEST_SHOW_UPDATE_MESSAGE";
export const REQUEST_SET_DOWNLOAD = "REQUEST_SET_DOWNLOAD";
export const REQUEST_SET_STATUS = "REQUEST_SET_STATUS";
export const REQUEST_START_PRINT = "REQUEST_START_PRINT";
export const REQUEST_STOP_PRINT = "REQUEST_STOP_PRINT";

// Locale Actions
export const REQUEST_SET_LOCALE = "REQUEST_SET_LOCALE";
export const REQUEST_SET_LOCALE_SUCCESS = "REQUEST_SET_LOCALE_SUCCESS";

// Spinner Actions
export const REQUEST_SPINNER_START = "REQUEST_SPINNER_START";
export const REQUEST_SPINNER_STOP = "REQUEST_SPINNER_STOP";

// Job Actions

export const REQUEST_ACTIVE_JOBS = "REQUEST_ACTIVE_JOBS";
export const REQUEST_SET_NEW_CRIME_TYPE_SUCCESS =
  "REQUEST_SET_NEW_CRIME_TYPE_SUCCESS";
export const REQUEST_ACTIVE_JOBS_SUCCESS = "REQUEST_ACTIVE_JOBS_SUCCESS";
export const REQUEST_ACTION_START = "REQUEST_ACTION_START";
export const REQUEST_ACTIVE_JOBS_FAILED = "REQUEST_ACTIVE_JOBS_FAILED";
export const REQUEST_CREATE_JOB = "REQUEST_CREATE_JOB";
export const REQUEST_CREATE_JOB_SUCCESS = "REQUEST_CREATE_JOB_SUCCESS";
export const REQUEST_CREATE_JOB_FAILED = "REQUEST_CREATE_JOB_FAILED";
export const REQUEST_ACTION_HISTORY = "REQUEST_ACTION_HISTORY";
export const REQUEST_ACTION_HISTORY_SUCCESS = "REQUEST_ACTION_HISTORY_SUCCESS";
export const REQUEST_ACTION_HISTORY_FAILED = "REQUEST_ACTION_HISTORY_FAILED";
export const REQUEST_REMOVE_JOB = "REQUEST_REMOVE_JOB";
export const REQUEST_REMOVE_JOB_SUCCESS = "REQUEST_REMOVE_JOB_SUCCESS";
export const REQUEST_REMOVE_JOB_FAILED = "REQUEST_REMOVE_JOB_FAILED";
export const REQUEST_NEW_ACTION = "REQUEST_NEW_ACTION";
export const REQUEST_NEW_ACTION_SUCCESS = "REQUEST_NEW_ACTION_SUCCESS";
export const REQUEST_NEW_ACTION_FAILED = "REQUEST_NEW_ACTION_FAILED";
export const REQUEST_SUBMIT_PRINTS = "REQUEST_SUBMIT_PRINTS";
export const REQUEST_SUBMIT_PRINTS_SUCCESS = "REQUEST_SUBMIT_PRINTS_SUCCESS";
export const REQUEST_SUBMIT_PRINTS_FAILED = "REQUEST_SUBMIT_PRINTS_FAILED";
export const REQUEST_SAVE_CARD_SCAN_BOXES = "REQUEST_SAVE_CARD_SCAN_BOXES";
export const REQUEST_ASC_SORT = "REQUEST_ASC_SORT";
export const REQUEST_REMOVE_FILTER = "REQUEST_REMOVE_FILTER";
export const REQUEST_SHOW_FILTERED_DATA = "REQUEST_SHOW_FILTERED_DATA";
export const REQUEST_ACTION_SUCCESS = "REQUEST_ACTION_SUCCESS";
export const REQUEST_OPEN_JOB = "REQUEST_OPEN_JOB";
export const REQUEST_SHOW_JOB_FILTERED_DATA = "REQUEST_SHOW_JOB_FILTERED_DATA";
export const REQUEST_HISTORY_SORT = "REQUEST_HISTORY_SORT";
export const REQUEST_CLOSE_FILTER = "REQUEST_CLOSE_FILTER";
export const REQUEST_WILDCARD_SEARCH = "REQUEST_WILDCARD_SEARCH";
export const REQUEST_SET_WILDCARD_SEARCH = "REQUEST_SET_WILDCARD_SEARCH";
export const REQUEST_CLOSE_FILTER_SUCCESS = "REQUEST_CLOSE_FILTER_SUCCESS";
export const REQUEST_CLOSE_ACTION_FILTER = "REQUEST_CLOSE_ACTION_FILTER";
export const REQUEST_SET_WILDCARD_ACTION_SEARCH = "REQUEST_SET_WILDCARD_ACTION_SEARCH";
export const REQUEST_CLOSE_ACTION_FILTER_SUCCESS =
  "REQUEST_CLOSE_ACTION_FILTER_SUCCESS";
export const REQUEST_FILTERED_ACTIVE_JOBS_SUCCESS =
  "REQUEST_FILTERED_ACTIVE_JOBS_SUCCESS";
export const REQUEST_UPDATE_JOB = "REQUEST_UPDATE_JOB";
export const REQUEST_UPDATE_JOB_SUCCESS = "REQUEST_UPDATE_JOB_SUCCESS";
export const REQUEST_UPDATE_JOB_FAILED = "REQUEST_UPDATE_JOB_FAILED";
export const REQUEST_CLEAR_TRANSACTION_HISTORY = "REQUEST_CLEAR_TRANSACTION_HISTORY";

export const SET_JOB_TYPE_FILTER_APPLIED = "SET_JOB_TYPE_FILTER_APPLIED";
export const REQUEST_JOB_AUDIT_LOGS = "REQUEST_JOB_AUDIT_LOGS";
export const REQUEST_JOB_AUDIT_LOGS_SUCESS = "REQUEST_JOB_AUDIT_LOGS_SUCESS";
export const REQUEST_SET_BIOMETRICS_DETAILS = "REQUEST_SET_BIOMETRICS_DETAILS";
export const REQUEST_MATCHED_PERSON_CHANGED = "REQUEST_MATCHED_PERSON_CHANGED";
export const RECIVED_MATCHED_PERSON_CHANGED = "RECIVED_MATCHED_PERSON_CHANGED";
export const REQUEST_SAVE_ANNOTATIONS = "REQUEST_SAVE_ANNOTATIONS";
export const REQUEST_SET_CIVIL_SEARCH = "REQUEST_SET_CIVIL_SEARCH";
export const REQUEST_SET_PAGE = "REQUEST_SET_PAGE";
export const REQUEST_START_SCAN_TYPE = "REQUEST_START_SCAN_TYPE";
export const REQUEST_SET_AUDIT_LOGS_FILTER = "REQUEST_SET_AUDIT_LOGS_FILTER";
export const REQUEST_RESET_AUDIT_LOGS_FILTER = "REQUEST_RESET_AUDIT_LOGS_FILTER";
export const REQUEST_AUDIT_LOG_SORT = "REQUEST_AUDIT_LOG_SORT";
export const REQUEST_SET_AUDIT_LOG_SORT = "REQUEST_SET_AUDIT_LOG_SORT";
export const REQUEST_RESET_AUDIT_LOG_SORT = "REQUEST_RESET_AUDIT_LOG_SORT";

// Modal Actions
export const REQUEST_SHOW_MODAL = "REQUEST_SHOW_MODAL";
export const REQUEST_HIDE_MODAL = "REQUEST_HIDE_MODAL";

// Global Actions
export const REQUEST_INPUT_FIELD_CHANGED = "REQUEST_INPUT_FIELD_CHANGED";
export const REQUEST_INPUT_FIELD_CHANGED_FAILED =
  "REQUEST_INPUT_FIELD_CHANGED_FAILED";
export const RECEIVE_INPUT_FIELD_CHANGED = "RECEIVE_INPUT_FIELD_CHANGED";
export const RECEIVE_INPUT_FIELD_CHANGED_FAILED =
  "RECEIVE_INPUT_FIELD_CHANGED_FAILED";
export const REQUEST_PAGE_CHANGED = "REQUEST_PAGE_CHANGED";
export const REQUEST_PAGE_CHANGED_FAILED = "REQUEST_PAGE_CHANGED_FAILED";
export const REQUEST_TABLE_ROW_CLICKED = "REQUEST_TABLE_ROW_CLICKED";
export const RECEIVE_TABLE_ROW_CLICKED = "RECEIVE_TABLE_ROW_CLICKED";
export const REQUEST_TABLE_ROW_CLICKED_FAILED =
  "REQUEST_TABLE_ROW_CLICKED_FAILED";
export const REQUEST_CLEAR_LATENT_EDITOR_DATA =
  "REQUEST_CLEAR_LATENT_EDITOR_DATA";

// Notification Actions
export const REQUEST_SHOW_NOTIFICATION = "REQUEST_SHOW_NOTIFICATION";
export const REQUEST_REMOVE_NOTIFICATION = "REQUEST_REMOVE_NOTIFICATION";

// Scanner Actions
export const REQUEST_START_CARD_SCAN = "REQUEST_START_CARD_SCAN";
export const REQUEST_START_CARD_SCAN_SUCCESS =
  "REQUEST_START_CARD_SCAN_SUCCESS";
export const REQUEST_START_CARD_SCAN_FAILED = "REQUEST_START_CARD_SCAN_FAILED";
export const REQUEST_SEGMENTED_CARD_SCAN = "REQUEST_SEGMENTED_CARD_SCAN";
export const REQUEST_SEGMENTED_CARD_SCAN_SUCCESS =
  "REQUEST_SEGMENTED_CARD_SCAN_SUCCESS";
export const REQUEST_SEGMENTED_CARD_SCAN_FAILED =
  "REQUEST_SEGMENTED_CARD_SCAN_FAILED";
export const REQUEST_FETCH_SCAN_SOURCES = "REQUEST_FETCH_SCAN_SOURCES";
export const REQUEST_FETCH_SCAN_SOURCES_SUCCESS =
  "REQUEST_FETCH_SCAN_SOURCES_SUCCESS";
export const REQUEST_FETCH_SCAN_SOURCES_FAILED =
  "REQUEST_FETCH_SCAN_SOURCES_FAILED";

export const REQUEST_START_SCANNER = "REQUEST_START_SCANNER";
export const REQUEST_START_SCANNER_SUCCESS = "REQUEST_START_SCANNER_SUCCESS";
export const REQUEST_START_SCANNER_FAILED = "REQUEST_START_SCANNER_FAILED";
export const REQUEST_STOP_SCANNER = "REQUEST_STOP_SCANNER";
export const REQUEST_STOP_SCANNER_SUCCESS = "REQUEST_STOP_SCANNER_SUCCESS";
export const REQUEST_STOP_SCANNER_FAILED = "REQUEST_STOP_SCANNER_FAILED";

//Latent Flow Actions
export const REQUEST_RECEIVED_IMAGE = "REQUEST_RECEIVED_IMAGE";
export const REQUEST_SAVE_LATENT_EDITED_IMAGE =
  "REQUEST_SAVE_LATENT_EDITED_IMAGE";
export const REQUEST_SAVE_SUD = "REQUEST_SAVE_SUD";
export const REQUEST_SAVE_SUD_SUCCESS = "REQUEST_SAVE_SUD_SUCCESS";
export const REQUEST_SHOW_SCANNED_IMAGE = "REQUEST_SHOW_SCANNED_IMAGE";
export const REQUEST_IMPORT_IMAGE = "REQUEST_IMPORT_IMAGE";
export const REQUEST_START_LATENT_SCAN = "REQUEST_START_LATENT_SCAN";
export const REQUEST_STOP_LATENT_SCAN = "REQUEST_STOP_LATENT_SCAN";
//Tenprint Flow Actions

export const REQUEST_TENPRINT_VERIFY = "REQUEST_TENPRINT_VERIFY";
export const REQUEST_SET_TENPRINT_VERIFY = "REQUEST_SET_TENPRINT_VERIFY";
export const REQUEST_SET_TENPRINT_VERIFY_HAND =
  "REQUEST_SET_TENPRINT_VERIFY_HAND";
export const REQUEST_SET_TENPRINT_VERIFY_STORE =
  "REQUEST_SET_TENPRINT_VERIFY_STORE";
export const REQUEST_SET_TENPRINT_VERIFY_MATCH =
  "REQUEST_SET_TENPRINT_VERIFY_MATCH";
export const REQUEST_SET_TENPRINT__VERIFY_MATCHED_DATA =
  "REQUEST_SET_TENPRINT__VERIFY_MATCHED_DATA";
export const REQUEST_REMOVE_TENPRINT__VERIFY_MATCHED_DATA =
  "REQUEST_REMOVE_TENPRINT__VERIFY_MATCHED_DATA";
export const REQUEST_SHOW_ADJUDICATOR = "REQUEST_SHOW_ADJUDICATOR";
export const REQUEST_SHOW_ADJUDICATOR_SUCCESS =
  "REQUEST_SHOW_ADJUDICATOR_SUCCESS";
export const REQUEST_MODAL_DATA = "REQUEST_MODAL_DATA";
export const RECIVED_MODAL_DATA = "RECIVED_MODAL_DATA";
export const REQUEST_HIDE_ADJUDICATOR = "REQUEST_HIDE_ADJUDICATOR";
export const RECIEVE_NOTE_STORE = "RECIEVE_NOTE_STORE";
export const MODAL_CHECKBOX = "MODAL_CHECKBOX";
export const REQUEST_SET_ADD_NEW = "REQUEST_SET_ADD_NEW";
export const REQUEST_DELETE_NEW_CRIME_DATA = "REQUEST_DELETE_NEW_CRIME_DATA";
export const RESET_ACTION_TYPE = "RESET_ACTION_TYPE";
export const REQUEST_RESET_CRIME_DATA = "REQUEST_RESET_CRIME_DATA";

//tenprint verify new
export const REQUEST_SELECTED_TENPRINT_TYPE = "REQUEST_SELECTED_TENPRINT_TYPE";

//Live Scan Flow Action
export const REQUEST_SELECT_FINGER = "REQUEST_SELECT_FINGER";
export const REQUEST_SELECT_FINGER_SUCCESS = "REQUEST_SELECT_FINGER_SUCCESS";
export const REQUEST_START_SCAN = "REQUEST_START_SCAN";
export const REQUEST_STOP_SCAN = "REQUEST_STOP_SCAN";
export const REQUEST_STORE_INITIAL_LIVESCAN_DATA =
  "REQUEST_STORE_INITIAL_LIVESCAN_DATA";
export const REQUEST_STORE_COMPLETED_DATA = "REQUEST_STORE_COMPLETED_DATA";
export const REQUEST_STORE_COMPLETED_DATA_SUCCESS =
  "REQUEST_STORE_COMPLETED_DATA_SUCCESS";
export const REQUEST_SAVE_LIVESCAN_FINGERPRINTS =
  "REQUEST_SAVE_LIVESCAN_FINGERPRINTS";
export const REQUEST_ANNOTATION_SELECT_CHANGED =
  "REQUEST_ANNOTATION_SELECT_CHANGED";
export const REQUEST_ANNOTATION_NOTES_CHANGED =
  "REQUEST_ANNOTATION_NOTES_CHANGED";
export const REQUEST_ANNOTATION_FINGER_SAVE = "REQUEST_ANNOTATION_FINGER_SAVE";
export const REQUEST_REMOVE_ANNOTATED_FINGERS =
  "REQUEST_REMOVE_ANNOTATED_FINGERS";
export const REQUEST_CAPTURE_DATA = "REQUEST_CAPTURE_DATA";
export const REQUEST_MUGSHOT_DATA = "REQUEST_MUGSHOT_DATA";
export const REQUEST_LOAD_LIVESCAN = "REQUEST_LOAD_LIVESCAN";
export const REQUEST_CLEAR_LIVESCAN_DATA = "REQUEST_CLEAR_LIVESCAN_DATA";
export const REQUEST_STORE_CANON_DATA = "REQUEST_STORE_CANON_DATA";
export const REQUEST_CAPTURE_DATA_CANON = "REQUEST_CAPTURE_DATA_CANON";
export const REQUEST_REMOVE_CANON_DATA = "REQUEST_REMOVE_CANON_DATA";
export const REQUEST_SCANNED_TYPE = "REQUEST_SCANNED_TYPE";
export const REQUEST_START_CANON_LIVE_VIEW = "REQUEST_START_CANON_LIVE_VIEW";
export const REQUEST_STOP_CANON_LIVE_VIEW = "REQUEST_STOP_CANON_LIVE_VIEW";
export const REQUEST_ClOSE_LIVESCAN_PROGRESS =
  "REQUEST_ClOSE_LIVESCAN_PROGRESS";

//AuditLog flow
export const REQUEST_SHOW_AUDITLOG = "REQUEST_SHOW_AUDITLOG";
export const REQUEST_SHOW_AUDITLOG_SUCCESS = "REQUEST_SHOW_AUDITLOG_SUCCESS";
export const REQUEST_AUDITLOG_PAGES = "REQUEST_AUDITLOG_PAGES";
//Scanned Biometrics
export const REQUEST_SCANNED_IMAGE_MODAL = "REQUEST_SCANNED_IMAGE_MODAL";
export const REQUEST_SELECTED_BIOMETRICS = "REQUEST_SELECTED_BIOMETRICS";
export const REQUEST_BIOMETRICS_MUGSHOT = "REQUEST_BIOMETRICS_MUGSHOT";
export const RECEIVED_BIOMETRICS_MUGSHOT = "RECEIVED_BIOMETRICS_MUGSHOT";
export const REQUEST_REMOVE_BIOMETRIC_MUGSHOT =
  "REQUEST_REMOVE_BIOMETRIC_MUGSHOT";
export const REQUEST_UPDATE_JOB_BIOMETRICS = "REQUEST_UPDATE_JOB_BIOMETRICS";

// costom search Actions

export const REQUEST_SET_PERSON_DATA = "REQUEST_SET_PERSON_DATA";
export const REQUEST_RESET_CUSTOM_SEARCH_DATA =
  "REQUEST_RESET_CUSTOM_SEARCH_DATA";
export const REQUEST_SET_TENPRINT_VERIFY_RAW =
  "REQUEST_SET_TENPRINT_VERIFY_RAW";
export const REQUEST_SET_CUSTOM_SEARCH_ID = "REQUEST_SET_CUSTOM_SEARCH_ID";
export const REQUEST_UPDATE_SEARCH_TEXT = "REQUEST_UPDATE_SEARCH_TEXT";
export const REQUEST_UPDATE_SEARCH_TEXT_SUCCESS =
  "REQUEST_UPDATE_SEARCH_TEXT_SUCCESS";
export const REQUEST_GET_JOB_SEARCH_TEXT = "REQUEST_GET_JOB_SEARCH_TEXT";
export const REQUEST_GET_JOB_SEARCH_TEXT_SUCCESS =
  "REQUEST_GET_JOB_SEARCH_TEXT_SUCCESS";
export const REQUEST_SET_CUSTOM_SEARCH_DATABASE =
  "REQUEST_SET_CUSTOM_SEARCH_DATABASE";

// palm scan
export const PALM_SCAN_WORKFLOW = "PALM_SCAN_WORKFLOW";
export const LIVE_SCAN_WORKFLOW = "LIVE_SCAN_WORKFLOW";
export const TOGGLE_PANE_VISIBILITY = "TOGGLE_PANE_VISIBILITY";

// App Action Types
export const UPDATE_SERVER_STATUS = "UPDATE_SERVER_STATUS";

export const UPDATE_NFIQ_QUALITY_THRESHOLD = "UPDATE_NFIQ_QUALITY_THRESHOLD";
export const REQUEST_NFIQ_QUALITY_THRESHOLD = "REQUEST_NFIQ_QUALITY_THRESHOLD";
export const REQUEST_NFIQ_QUALITY_THRESHOLD_SUCCESS =
  "REQUEST_NFIQ_QUALITY_THRESHOLD_SUCCESS";

export const REQUEST_SAVE_LIVESCAN_FINGERPRINTS_RESUMABLE =
  "REQUEST_SAVE_LIVESCAN_FINGERPRINTS_RESUMABLE";
export const REQUEST_SAVE_LIVESCAN_FINGERPRINTS_RESUMABLE_SUCCESS =
  "REQUEST_SAVE_LIVESCAN_FINGERPRINTS_RESUMABLE_SUCCESS";

// card scan
export const CARD_SCAN_WORKFLOW = "CARD_SCAN_WORKFLOW";
export const CARD_SCAN_RESET = "CARD_SCAN_RESET";

// export const REQUEST_START_CARDSCAN = "REQUEST_START_CARDSCAN";
// export const REQUEST_START_CARDSCAN_SUCCESS = "REQUEST_START_CARDSCAN_SUCCESS";
// export const REQUEST_START_CARDSCAN_FAILED = "REQUEST_START_CARDSCAN_FAILED";
// export const REQUEST_FETCH_CARDSCAN_SOURCES = "REQUEST_FETCH_CARDSCAN_SOURCES";

// export const REQUEST_RESET_CARD_SCAN_BOXES = "REQUEST_RESET_CARD_SCAN_BOXES";
export const REQUEST_CHECK_FOR_APP_UPDATE = "REQUEST_CHECK_FOR_APP_UPDATE";
export const REQUEST_LOGIN_POPUP = "REQUEST_LOGIN_POPUP";
export const REQUEST_SET_LOOKUPS = "REQUEST_SET_LOOKUPS";
export const REQUEST_STORE_LINK_DATA = "REQUEST_STORE_LINK_DATA";
export const REQUEST_RESET_LINK_DATA = "REQUEST_RESET_LINK_DATA";
export const REQUEST_SAVE_ANNOTATIONS_CUSTOM = "REQUEST_SAVE_ANNOTATIONS_CUSTOM";
export const REQUEST_RESET_PALMSCAN_ANNOTATIONS = "REQUEST_RESET_PALMSCAN_ANNOTATIONS";
export const REQUEST_UNMATCH = "REQUEST_UNMATCH";

//New Permistion api section.
export const REQUEST_SYSTEM_PERMISSION = "REQUEST_SYSTEM_PERMISSION";
export const REQUEST_SYSTEM_PERMISSION_SUCCESS = "REQUEST_SYSTEM_PERMISSION_SUCCESS";
export const REQUEST_SYSTEM_PERMISSION_ERROR = "REQUEST_SYSTEM_PERMISSION_ERROR";
export const REQUEST_SET_LOOKUP_CRIME_TYPES = "REQUEST_SET_LOOKUP_CRIME_TYPES";
export const REQUEST_SET_CUSTOM_SEARCH_RESPONSE = "REQUEST_SET_CUSTOM_SEARCH_RESPONSE";
export const REQUEST_SAVE_INNOV_SPINNER = "REQUEST_SAVE_INNOV_SPINNER";
export const REQUEST_SET_LOOKUP_NATIONALITY = "REQUEST_SET_LOOKUP_NATIONALITY";
export const REQUEST_SAVE_INNOV_LICENSE = "REQUEST_SAVE_INNOV_LICENSE";
export const REQUEST_SET_USER_INFO = "REQUEST_SET_USER_INFO";
export const REQUEST_SHOW_MODAL_WITH_DATA = "REQUEST_SHOW_MODAL_WITH_DATA";