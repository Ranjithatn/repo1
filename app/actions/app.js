import {
  UPDATE_SERVER_STATUS,
  REQUEST_NFIQ_QUALITY_THRESHOLD,
  REQUEST_NFIQ_QUALITY_THRESHOLD_SUCCESS,
} from "./actionTypes";

export const requestUpdateServerStatus = (data) => {
	return {
	  type: UPDATE_SERVER_STATUS,
	  payload: data
	};
}

export const requestNFIQQualityThreshold = (data) => {
	return {
	  type: REQUEST_NFIQ_QUALITY_THRESHOLD,
	  payload: data
	};
}

export const requestUpdateNFIQQualityThreshold = (data) => {
	return {
	  type: REQUEST_NFIQ_QUALITY_THRESHOLD_SUCCESS,
	  payload: data
	};
}

