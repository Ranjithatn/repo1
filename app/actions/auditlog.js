// requestShowAuditLog
import {
  REQUEST_SHOW_AUDITLOG,
  REQUEST_SHOW_AUDITLOG_SUCCESS,
  REQUEST_AUDITLOG_PAGES,
  REQUEST_JOB_AUDIT_LOGS,
  REQUEST_JOB_AUDIT_LOGS_SUCESS,
  REQUEST_SET_AUDIT_LOGS_FILTER,
  REQUEST_RESET_AUDIT_LOGS_FILTER,
  REQUEST_AUDIT_LOG_SORT,
  REQUEST_SET_AUDIT_LOG_SORT,
  REQUEST_RESET_AUDIT_LOG_SORT
} from "./actionTypes";

export const requestShowAuditLog = (id, status, startDate, endDate, Pages) => {
  const data = {
    jobID: id,
    jobStatus: status,
    jobStartDate: startDate,
    jobEndDate: endDate,
    PageDetails: Pages
  };

  return {
    type: REQUEST_SHOW_AUDITLOG,
    payload: data
  };
};
export const requestOpenAudiLogSuccess = AuditlogData => ({
  type: REQUEST_SHOW_AUDITLOG_SUCCESS,
  payload: AuditlogData
});

type pageSettings = {
  pageNo: number | string
};

// export const requestAuditLogPages = (payload: ?pageSettings) => {
//   type: REQUEST_AUDITLOG_PAGES;
// };

export const requestJobAuditLogs = JobID => ({
  type: REQUEST_JOB_AUDIT_LOGS,
  payload: JobID
});

export const requestJobAuditLogsSucess = LogDetails => ({
  type: REQUEST_JOB_AUDIT_LOGS_SUCESS,
  payload: LogDetails
});
export const requestResetAditlogFilter = () => ({
  type: REQUEST_RESET_AUDIT_LOGS_FILTER
});

export const requestSetAuditLogFilter = (id, status, startDate, endDate, Pages) => {
  const data = {
    jobID: id,
    jobStatus: status,
    jobStartDate: startDate,
    jobEndDate: endDate,
    PageDetails: Pages
  };

  return {
    type: REQUEST_SET_AUDIT_LOGS_FILTER,
    payload: data
  };
};
export const requrstAuditLogSort = (column, type,page) => {
  const data = {
    column: column,
    type: type,
    page:page
  
  };

  return {
    type: REQUEST_AUDIT_LOG_SORT,
    payload: data
  };
};
export const requestSetAuditlogSort = (column, type,) => {
  const data = {
    column: column,
    type: type
  
  };

  return {
    type: REQUEST_SET_AUDIT_LOG_SORT,
    payload: data
  };
};
export const requestResetAuditlogSort = () => ({
 
    type: REQUEST_RESET_AUDIT_LOG_SORT
 
});
