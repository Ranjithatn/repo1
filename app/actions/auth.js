// @flow
import {
  REQUEST_LOGIN,
  REQUEST_LOGIN_SUCCESS,
  REQUEST_LOGIN_FAILED,
  REQUEST_LOGOUT,
  REQUEST_LOGOUT_SUCCESS,
  REQUEST_SET_NEW_UPDATE,
  REQUEST_SHOW_UPDATE_MESSAGE,
  REQUEST_SET_DOWNLOAD,
  REQUEST_SET_STATUS,
  REQUEST_CHECK_FOR_APP_UPDATE,
  REQUEST_LOGIN_POPUP,
  REQUEST_SET_LOOKUPS,
  REQUEST_SET_LOOKUP_CRIME_TYPES,
  REQUEST_SET_LOOKUP_NATIONALITY,
  REQUEST_SET_USER_INFO,
} from "./actionTypes";

type credentials = {
  username?: string,
  password?: string
};

export const requestLogin = (payload: credentials, history) => {
  return {
    type: REQUEST_LOGIN,
    payload,
    history
  };
};


export const requestLoginPopup = (payload: credentials) => {
  return {
    type: REQUEST_LOGIN_POPUP,
    payload,
  };
};


export const requestLoginSuccess = (authData: {
  jwt: string,
  roles: any,
  parsedRoles: array,
}) => {
  return {
    type: REQUEST_LOGIN_SUCCESS,
    payload: authData
  };
};

export const requestLoginFailed = (error: string) => {
  return {
    type: REQUEST_LOGIN_FAILED,
    payload: error
  };
};

export const requestLogout = history => {
  return {
    type: REQUEST_LOGOUT,
    history
  };
};

export const requestLogoutSuccess = () => {
  return {
    type: REQUEST_LOGOUT_SUCCESS
  };
};
export const requestSetNewUpdate = () => {
  return {
    type: REQUEST_SET_NEW_UPDATE
  };
};
export const requestSetDownload = version => {
  return {
    type: REQUEST_SET_DOWNLOAD,
    payload: version
  };
};
export const requestShowUpdateMessage = msg => {
  return {
    type: REQUEST_SHOW_UPDATE_MESSAGE,
    payload: msg
  };
};
export const requestSetStatus = status => {
  return {
    type: REQUEST_SET_STATUS,
    payload: status
  };
};



export const requestCheckForAppUpdate = () => {
  return {
    type: REQUEST_CHECK_FOR_APP_UPDATE,
  };
};

export const requestSetLookups = (data) => {
  return {
    type: REQUEST_SET_LOOKUPS,
    payload: data,
  }
}

export const requestSetLookupCrimeTypes = (data) => {
  return {
    type: REQUEST_SET_LOOKUP_CRIME_TYPES,
    payload: data,
  }
}

export const requestSetLookupNationality = (data) => {
  return {
    type: REQUEST_SET_LOOKUP_NATIONALITY,
    payload: data,
  }
}


export const requestSetUserInfo = (data) => {
  return {
    type: REQUEST_SET_USER_INFO,
    payload: data || {},
  }
}

