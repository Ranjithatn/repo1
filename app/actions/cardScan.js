// @flow
import {
  CARD_SCAN_WORKFLOW,
  CARD_SCAN_RESET,
} from "./actionTypes";

export const cardScanWorkflow = (data) => ({
  type: CARD_SCAN_WORKFLOW,
  payload: data,
});

export const cardScanReset = (data) => ({
  type: CARD_SCAN_RESET,
  payload: data,
});

