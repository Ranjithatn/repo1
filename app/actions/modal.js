// @flow
import { REQUEST_SHOW_MODAL, REQUEST_HIDE_MODAL, REQUEST_SHOW_MODAL_WITH_DATA } from "./actionTypes";

export const requestShowModal = payload => {
  return {
    type: REQUEST_SHOW_MODAL,
    payload
  };
};

export const requestHideModal = () => ({
  type: REQUEST_HIDE_MODAL
});


export const requestShowModalWithData = payload => {
  return {
    type: REQUEST_SHOW_MODAL_WITH_DATA,
    payload
  };
};
