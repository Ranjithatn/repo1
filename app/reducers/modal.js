import { REQUEST_SHOW_MODAL, REQUEST_HIDE_MODAL } from "../actions/actionTypes";

const initialState = {
  modalType: "",
  modalProps: {}
};

export default function modal(state = initialState, action = {}) {
  switch (action.type) {
    case REQUEST_SHOW_MODAL:
      const { modalType, modalProps } = action.payload;
      return { ...state, modalType: modalType, modalProps: modalProps, latentId: action.payload.latentId || '' };
    case REQUEST_HIDE_MODAL:
      return { modalType: "", modalProps: {} };
    default:
      return state;
  }
}
