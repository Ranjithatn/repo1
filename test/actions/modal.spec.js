import { requestShowModal, requestHideModal } from "../../app/actions/modal";
import { REQUEST_SHOW_MODAL, REQUEST_HIDE_MODAL } from "../../app/actions/actionTypes";

describe("modal actions", () => {
  let output;
  let expected;
  it("should create show modal action", () => {
    const payload = {
      modalType: "ADD_JOB",
      modalProps: {}
    };
    output = requestShowModal(payload);
    expected = {
      type: REQUEST_SHOW_MODAL,
      payload: {
        modalType: "ADD_JOB",
        modalProps: {}
      }
    };
    expect(output).toEqual(expected);
    expect(output).toMatchSnapshot();
  });

  it("should create hide modal action", () => {
    output = requestHideModal();
    expected = {
      type: REQUEST_HIDE_MODAL
    };
    expect(output).toEqual(expected);
    expect(output).toMatchSnapshot();
  });
});
