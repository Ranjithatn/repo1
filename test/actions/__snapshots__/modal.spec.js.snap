// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`modal actions should create hide modal action 1`] = `
Object {
  "type": "REQUEST_HIDE_MODAL",
}
`;

exports[`modal actions should create show modal action 1`] = `
Object {
  "payload": Object {
    "modalProps": Object {},
    "modalType": "ADD_JOB",
  },
  "type": "REQUEST_SHOW_MODAL",
}
`;
